tests/python/topi/python/test_topi_cumsum.py

```diff
@@ -40,10 +42,15 @@ def check_cumsum(np_ref, data, axis=None, dtype=None):
     check_cumsum(np.cumsum(data, dtype=np.int32), data)
     check_cumsum(np.cumsum(data), data, dtype="int64")
 
-    data = np.random.rand(10) > 0.5
-    check_cumsum(np.cumsum(data, dtype=np.int32), data, dtype="int32")
+    if target != "vulkan":
```

Comment:
this won't work because now target has some default attrs e.g. `vulkan -keys=vulkan,gpu -max_num_threads=256`

Reply:
Yes I also found that problem. Try running `pytest -k cumsum`. When run by pytest, target is always a string.